integration/spark/app/src/test/java/io/openlineage/spark/agent/ArgumentParserTest.java

```diff
@@ -159,6 +164,33 @@ void testConfToKinesisConfig() {
     assertEquals("test2", transportConfig.getProperties().get("test2"));
   }
 
+  @Test
+  void testLoadConfigFromEnvVars() {
+    try (MockedStatic mocked = mockStatic(Environment.class)) {
+      when(Environment.getAllEnvironmentVariables())
+          .thenReturn(Collections.singletonMap("OPENLINEAGE__TRANSPORT__TYPE", "console"));
+
+      SparkConf sparkConf = new SparkConf();
+      SparkOpenLineageConfig config = ArgumentParser.parse(sparkConf);
+      assertThat(config.getTransportConfig()).isInstanceOf(ConsoleConfig.class);
```

Comment:
isn't a console already a default transport? Would this test fail if environment variable not set?

Reply:
You're right, I'll change the test.